recipes/elfio/bld.bat

```diff
@@ -0,0 +1,3 @@
+cmake -DCMAKE_INSTALL_PREFIX=%PREFIX% -G Ninja -B _build
+
+cmake --install _build
```

Comment:
Can we run ctest on Windows too?

Reply:
No, the tests don't seem to build on Windows due to some boost linking errors. This is a header-only library anyway.